engine-precompiles/Cargo.toml

```diff
@@ -34,8 +33,8 @@ workspace = true
 
 [features]
 default = ["std"]
-std = ["aurora-engine-types/std", "aurora-engine-sdk/std", "blst", "bn/std", "aurora-evm/std", "ripemd/std", "sha2/std", "ethabi/std"]
-contract = ["aurora-engine-sdk/contract"]
+std = ["aurora-engine-types/std", "aurora-engine-sdk/std", "aurora-engine-sdk/bls", "bn/std", "aurora-evm/std", "ripemd/std", "sha2/std", "ethabi/std"]
+contract = ["aurora-engine-sdk/contract", "aurora-engine-sdk/bls"]
```

Comment:
Why is the `bls` feature needed for `contract`? I thought that feature was only if the standalone bls implementation should be included.

Reply:
The idea was not to include the heavy, specific `bls` logic in the `aurora-engine-sdk` by default. By providing the corresponding feature only.

Reply:
Our main idea is granularity. If the feature `contract` in `engine-sdk` is written as follows:
```
contract = "bls"
```
this means that `bls` will be included everywhere `engine-sdk.contract` is used, for example, in `engine`. However, this is redundant since the only place where it is actually used is `engine-precompiles`.

Reply:
Having a feature flag that removes everything related to bls does make more sense for external usage of the sdk crate. I was just worried about including unused code in the wasm binary, but maybe that is prevented by restricting the dependency according to compile target. Perhaps it is fine as-is.